models/blobs/ethereum/blobs_based_submitters.sql

```diff
@@ -0,0 +1,8 @@
+{{ config(
+    schema = 'blobs',
+    alias = 'based_submitters',
+    materialized = 'table',
+    file_format = 'delta'
+)}}
+
+SELECT evt_tx_hash AS tx_hash, 'Taiko' AS entity FROM {{ source('taikoxyz_ethereum', 'TaikoL1_evt_BlockProposed')}}
```

Comment:
can we include `block_number` as well? Best practice to include that in the join condition together with the tx_hash.